docs/source/index.rst

```diff
@@ -15,7 +15,10 @@ Welcome to the Cozmo SDK Beta!
     :maxdepth: 2
     :caption: Installation
 
-    install
+    initial
+    install-linux.rst
```

Comment:
The order of the install pages should be: macOS, Windows, and then Linux (same order as our OS usage percentages).